mu-plugins/build-processes-demo-features/build-processes-demo-features.php

```diff
@@ -23,17 +25,24 @@
 
 defined( 'ABSPATH' ) || exit;
 
-// Load the features plugin's translated strings.
-function bpd_features_load_textdomain() {
-	load_muplugin_textdomain( 'build-processes-demo-features', dirname( plugin_basename( __FILE__ ) ) . '/languages' );
-}
-add_action( 'init', 'bpd_features_load_textdomain' );
+// Define plugin constants.
+define( 'BPD_FEATURES_VERSION', '0.1.0' );
+define( 'BPD_FEATURES_DIR', plugin_dir_path( __FILE__ ) );
+define( 'BPD_FEATURES_URL', plugin_dir_url( __FILE__ ) );
 
-// Load the features plugin's files.
-foreach ( glob( __DIR__ . '/includes/*.php' ) as $bpd_features_filename ) {
-	if ( preg_match( '#/includes/_#i', $bpd_features_filename ) ) {
-		continue; // Ignore files prefixed with an underscore.
-	}
+// Define minimum environment requirements.
```

Comment:
Could these version definitions be pulled from the plugin headers? e.g https://developer.wordpress.org/reference/functions/get_plugin_data/

Reply:
Good idea! Never thought about doing a `get_plugin_data( __FILE__ )` but I don't see why not. I'll experiment with this.